wallet/src/index.tsx

```diff
@@ -5,7 +5,7 @@ import '@endo/eventual-send/shim';
 
 // Ambient types. Needed only for dev but this does a runtime import.
 import '@agoric/ertp/exported';
-import '@agoric/notifier/exported';
+import '@agoric/notifier/exported.js';
```

Comment:
consider updating the others too

Reply:
This was just to fix compilation. In notifier's package.json it names the export explicitly "exported.js", so it didn't work to just import "exported"

Reply:
yeah. I'm just thinking the others will need the same when they get to ESM and we should get into the habit of ESM-style imports